examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,139 @@
+"""
+Comparing connectomes on different reference atlases
+====================================================
+
+An important part of the example, is to turn a parcellation into
+connectome for visualization. This requires choosing centers for each parcel
+or network, via :func:`nilearn.plotting.find_parcellation_cut_coords` for
+parcellation based on labels and
+:func:`nilearn.plotting.find_probabilistic_atlas_cut_coords` for
+parcellation based on probabilistic values.
+
+In the intermediary steps, we make use of
+:class:`nilearn.input_data.NiftiLabelsMasker` and
+:class:`nilearn.input_data.NiftiMapsMasker` to extract time series from nifti
+objects using different parcellation atlases.
+The time series of all subjects of the ADHD Dataset are concatenated and
+given directly to :class:`nilearn.connectome.ConnectivityMeasure` for
+computing parcel-wise correlation matrices for each atlas across all subjects.
+
+Mean correlation matrix is displayed on glass brain on extracted coordinates.
+
+# author: Amadeus Kanaan
+
+"""
+# General imports
+import numpy as np
+
+####################################################################
+# Load atlases
+# -------------
+from nilearn import datasets
+
+destrieux = datasets.fetch_atlas_destrieux_2009()
+yeo = datasets.fetch_atlas_yeo_2011()
+harvard_oxford = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm')
+# with split into left and right hemispheres
+harvard_oxford_s = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm',
+                                                       symmetric_split=True)
+
+print('Destrieux atlas nifti image (3D) is located at: %s' % destrieux['maps'])
+print('Yeo atlas nifti image (3D) with 17 parcels and liberal mask is located '
+      'at: %s' % yeo['thick_17'])
+print('Harvard Oxford atlas nifti image (3D) thresholded at .25 is located '
+      'at: %s' % harvard_oxford['maps'])
+# Store atlases in a dictionary. Atlases are for extracting coordinates.
+atlases = {
+    'Destrieux Atlas (struct)': destrieux['maps'],
+    'Yeo Atlas 17 thick (func)': yeo['thick_17'],
+    'Harvard Oxford > 25% (struct one hemisphere)': harvard_oxford['maps'],
+    'Harvard Oxford > 25% (struct two hemispheres)': harvard_oxford_s['maps']}
```

Comment:
That one gives a connectome that is way too dense, and hard to look at. I think that we should remove it.

Reply:
We should probably also remove the Destrieux: that script takes 9mn to run on the CI. That's a lot.

Reply:
Actually, thinking of it, I am in favor of simplifying that script down all the way to having only 1 atlas of each kind. It is enough to convey the message. I would keep Yeo and MSDL.